src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1543,6 +1545,72 @@ public XMPPError onSessionInfo(JingleSession session, JingleIQ iq)
         return null;
     }
 
+    /**
+     * Handles 'session-terminate' received from the client.
+     *
+     * {@inheritDoc}
+     */
+    @Override
+    public XMPPError onSessionTerminate(JingleSession session, JingleIQ iq)
+    {
+        Participant participant = findParticipantForJingleSession(session);
+
+        // FIXME: (duplicate) there's very similar logic in onSessionAccept/onSessionInfo
+        if (participant == null)
+        {
+            String errorMsg = "No session for " + session.getAddress();
+
+            logger.warn("onSessionInfo: " + errorMsg);
```

Comment:
Log "onSessionTerminate" or something more useful? 

"No session" is also not accurate -- there's a Jingle session, but no participant associated with it.

Reply:
yes, thanks